src/lib.rs

```diff
@@ -193,8 +266,6 @@ impl<'a> From<Locator<'a>> for webdriver::command::LocatorParameters {
     }
 }
 
-pub use crate::session::Client;
```

Comment:
Why did this have to move?

Reply:
Sorry, I removed some time while doing something it that part of the file and then when I inserted it again I just did it at the top of the file like I do all imports, but I moved it now to where it's been